<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfazScrapping</class>
 <widget class="QMainWindow" name="InterfazScrapping">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Web Scrapping</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="fixWin10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabScrapping">
       <attribute name="title">
        <string>Scrapping</string>
       </attribute>
       <widget class="QPushButton" name="pBDescargar">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Descargar</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="tEURLPagina">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;https://www.tripadvisor.com/Restaurant_Review-g34439-d10170268-Reviews-Santorini_by_Georgios-Miami_Beach_Florida.html&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="lURL">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>321</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>URL de la pagina (de tripadvisor)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="pTEStatus">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>701</width>
          <height>401</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lResultado">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>126</y>
          <width>321</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Resultado&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabSentimientos">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Analisis de sentimientos</string>
       </attribute>
       <widget class="QLabel" name="lSeleccionar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>321</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Seleccionar CSV</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lEArchivo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pBArchivoBuscar">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar..</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pBAnalizar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Analizar</string>
        </property>
       </widget>
       <widget class="QTableView" name="tVResultado">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>701</width>
          <height>401</height>
         </rect>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashLine</enum>
        </property>
       </widget>
       <widget class="QLabel" name="lResultado_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>321</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Resultado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>1</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
